pen/static/pen/css/style.css

```diff
@@ -0,0 +1,79 @@
+.highlight
+{
+    border: 3px solid black;
+    font-weight: bold;
+    background-color: red;
+    height: 70px;
+    width: 200px;
+    font-size: 45px;
+    text-align: center;
+    border-radius: 50px;
+    border: none;
+    margin: 3px;
+    justify-content: center;
+    display: inline-block;
+    box-shadow: 5px 10px;
+}
+
+.highlighted
+{
+    border: 3px solid black;
+    font-weight: bold;
+    background-color: #00FF00;
+    height: 70px;
+    width: 200px;
+    font-size: 45px;
+    text-align: center;
+    border-radius: 50px;
+    border: none;
```

Comment:
dubel property border

Reply:
Line no. 28 of the code has been removed